garage/tf/q_functions/continuous_mlp_q_function.py

```diff
@@ -5,10 +5,10 @@
 import garage.tf.core.layers as L
 from garage.tf.core.layers import batch_norm
 from garage.tf.misc import tensor_utils
-from garage.tf.q_functions import QFunction
+from garage.tf.q_functions import NetworkBuilder
```

Comment:
Where is NetworkBuilder? Am I missing something here?

Reply:
oh I renamed QFunction to NetworkBuilder before
I will change that back

Reply:
Done